op-chain-ops/genesis/config_test.go

```diff
@@ -175,3 +175,14 @@ func TestUpgradeScheduleDeployConfig_ActivateForkAtOffset(t *testing.T) {
 		}
 	})
 }
+
+func TestUpgradeScheduleDeployConfig_ForkOrdinal(t *testing.T) {
+	for i, fork := range rollup.AllForks[1:] {
+		if fork == rollup.Interop {
+			return
```

Comment:
Should this be a `continue` instead of `return`?

Reply:
it needs to be a return - interop is the last fork and this method will panic if it goes off the end.

Reply:
Would it be more future proof to change it to:
```
func TestUpgradeScheduleDeployConfig_ForkOrdinal(t *testing.T) {
    forks := rollup.AllForks[1:len(rollup.AllForks)-1]
    for i, fork := range forks {
        var d UpgradeScheduleDeployConfig
        d.ActivateForkAtOffset(fork, 0)
        require.EqualValues(t, i, d.ForkOrdinal(1))
    }
}
```